man/kernel-install.xml

```diff
@@ -171,7 +171,7 @@
     <para><varname>KERNEL_INSTALL_MACHINE_ID=</varname> is set for the plugins to the desired <replaceable>MACHINE-ID</replaceable> –
     either 32 hexadecimal characters or the special value <literal>Default</literal>.</para>
     <para><varname>KERNEL_INSTALL_BOOT_ROOT=</varname> is set for the plugins to the root directory (mount point, usually) of the hierarchy
-    where boot-loader entries, kernel images, and associated resources should be placed.  Can be overriden by setting <varname>BOOT_ROOT=</varname>.</para>
+    where boot-loader entries, kernel images, and associated resources should be placed. Can be overridden by setting <varname>BOOT_ROOT=</varname>.</para>
```

Comment:
L169 above double-spaces the full stop, so I did as well

Reply:
don't think it was intentional above either